<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>154</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Общие настройки</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>318</width>
       <height>78</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Номер измерительного пункта:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="mpNumLineEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Папка для сохранения файлов с измерениями:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="saveDirLineEdit">
        <property name="text">
         <string>/home/</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="saveDirToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>154</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Настройки параметров COM-порта</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>441</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Выбор порта:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Скорость передачи:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Число бит данных:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Бит четности:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Стоп-бит:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="flowControlLabel">
        <property name="text">
         <string>Управление потоком:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>154</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Настройки параметров сетевого соединения</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>281</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IP-адрес ВЦ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="destIPLineEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="destPortLineEdit">
        <property name="text">
         <string>6180</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>461</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
